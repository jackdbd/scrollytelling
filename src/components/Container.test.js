import React from "react";
import { shallow, mount, render } from "enzyme";
import "../setupEnzymeTests";
import Container from "./Container";

describe("<Container />", () => {
  it("renders one <div>", () => {
    const wrapper = mount(<Container />);
    expect(wrapper.find("div")).toHaveLength(1);
  });
  it("matches the snapshot", () => {
    expect(<Container />).toMatchSnapshot();
  });
});
