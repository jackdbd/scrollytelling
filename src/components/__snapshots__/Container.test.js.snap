// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> matches the snapshot 1`] = `
<styled.div
  backgroundColor="#d3d3d3"
  fontFamily="Arial"
/>
`;
